contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -430,8 +430,10 @@ var BaseWorkspaceView = BaseView.extend({
 	},
 	move_content:function(move_collection, source){
 		var MoveView = require("edit_channel/move/views");
-		var list = this.get_selected(true);
-		move_collection = new Models.ContentNodeCollection(_.pluck(list, 'model'));
+		if (!move_collection){
```

Comment:
Just to make sure, does this affect the clipboard/edit page toolbar move functions?